compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/generators/FakeOverrideGenerator.kt

```diff
@@ -67,9 +67,22 @@ class FakeOverrideGenerator(
     }
 
     fun IrClass.getFakeOverrides(klass: FirClass<*>, realDeclarations: Collection<FirDeclaration>): List<IrDeclaration> {
+        val classLookupTag = klass.symbol.toLookupTag()
+        val realDeclarationNames = realDeclarations.mapNotNullTo(mutableSetOf()) l@{
+            val callableDeclaration = (it as? FirCallableMemberDeclaration<*>) ?: return@l null
```

Comment:
nit: this `l` label does not seem to make the code more readable. Consider removing it and just use `@mapNotNullTo` where needed below.